### ISSUE TITLE
no-describe-selectors does not work for cucumber step_definitions

### ISSUE THREAD

**Issue by jlin412** [alecxe/eslint-plugin-protractor#49]
Is there that we can support cucumber step_definitions as well for this rule?

**Comment by alecxe** [comment]
@jlin412 good idea. I was afraid I should not be sticking to jasmine/mocha framework style in the plugin that much. Do you think it would be better to generalize `no-describe-selectors` to also support `cucumber`, or create a separate rule (set of rules) specifically for `cucumber`?

Thanks!

**Comment by igniteram** [comment]
@alecxe I would suggest to have to separate set of rules  as there many contrasting features between `jasmine` and `cucumber` . Let me see if could send some PR's!

**Comment by alecxe** [comment]
@igniteram would really appreciate it. Let me know if you need any help or any adjustments to the current rules or exposed settings to support the new cucumber rules. Thank you.

**Comment by alecxe** [comment]
@igniteram by the way, you can ping me on [gitter](https://gitter.im/alecxe/eslint-plugin-protractor) whenever needed

**Comment by igniteram** [comment]
Sure @alecxe will do that 👍

**Comment by jlin412** [comment]
Just throw it out there that no-get-in-it
https://github.com/alecxe/eslint-plugin-protractor/blob/master/docs/rules/no-get-in-it.md
rule
doesn't apply in cucumber step definition though it is not as critical as
the other one.
I am happy to help test if there is a PR.

**Comment by alecxe** [comment]
@jlin412 gotcha, please disable the rules that don't apply to your particular case.
